workers/unity/Packages/io.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -91,8 +91,9 @@ public static void LaunchClient()
             {
                 command = "osascript";
                 commandArgs = $@"-e 'tell application ""Terminal""
+                                     set projectDir to ""{Common.SpatialProjectRootDir}""
                                      activate
-                                     do script ""cd {Common.SpatialProjectRootDir} && {Common.SpatialBinary} {commandArgs}""
+                                     do script ""cd "" & quoted form of projectDir & "" && {Common.SpatialBinary} {commandArgs}""
```

Comment:
I find it almost magic how "quoted form of X" is valid osascript

Reply:
Yeah its pretty wild